convex/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "forceConsistentCasingInFileNames": true,
     "allowSyntheticDefaultImports": true,
     "module": "ESNext",
-    "moduleResolution": "Node",
+    "moduleResolution": "bundler",
```

Comment:
Is this related to the `esbuild` config? fwiw, I got `convex-helpers` working only after using regular `tsc` and explicit exports for every module that uses an index.ts.